### DISCUSSION TITLE
Remove bottom-0 on responsive breakpoint

### DISCUSSION THREAD

**Discussion by jochemgruter** [tailwindlabs/tailwindcss#5290] [Help] [answered]
Hi I need some help. In my div for mobile design I have the class `bottom-0`. But for larger screens this class shouldn't be applied. How can I do this?

Normally I would use another class, like with border:
`class="border sm:border-none"`

But how do this for `bottom-0`?

**Comment by ikuuAlex** [comment, marked as answer]
bottom-auto

**Reply by jochemgruter** [reply]
Thanks
